drivers/scheduler/k8s/k8s.go

```diff
@@ -125,7 +125,7 @@ const (
 const (
 	k8sNodeReadyTimeout    = 5 * time.Minute
 	volDirCleanupTimeout   = 5 * time.Minute
-	k8sObjectCreateTimeout = 2 * time.Minute
+	k8sObjectCreateTimeout = 4 * time.Minute
```

Comment:
Is this just "how long to wait for pods to be ready", that kind of thing?

Are we sure that we need to increase the timeout and we aren't covering up some other issue? Or is it just like... statefulsets starting 1 at a time and that takes too long?

Reply:
Yes Its about statefulsets only. About creation of k8s object.

Reply:
Thanks @Pure-AdamuKaapan I have updated code accordingly, Added new timeout variable for CSI clone and snapshot functionality.